app/edge/views/edge-jobs/edgeJob/edgeJob.html

```diff
@@ -7,7 +7,7 @@
         <uib-tabset active="$ctrl.state.activeTab">
           <uib-tab index="0" select="$ctrl.showEditor()">
             <uib-tab-heading>
-              <span><pr-icon icon="'tool'"></pr-icon></span> Configuration
+              <span><pr-icon icon="'wrench'"></pr-icon></span> Configuration
```

Comment:
Not sure if intentional, but this change is not present in the EE pr.

Reply:
updated EE